huaweicloud/services/vpc/resource_huaweicloud_vpc.go

```diff
@@ -63,6 +63,16 @@ func ResourceVirtualPrivateCloudV1() *schema.Resource {
 				Optional:     true,
 				ValidateFunc: utils.ValidateCIDR,
 			},
```

Comment:
add a special description, then you can delete `secondary_cidr` in docs.
```suggestion
				Optional:     true,
				ValidateFunc: utils.ValidateCIDR,
				Description:  "schema: Deprecated; use secondary_cidrs instead",
			},
```